src/questions/intro-to-lending-borrowing.yaml

```diff
@@ -1,19 +1,402 @@
+  answerKeys:
+    - C
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Over-collateralization helps to find bugs in the smart-contracts.
+      key: A
+    - content: Over-collateralization helps to increase the interest rates for lenders.
+      key: B
+    - content: Over-collateralization is needed to cover potential losses in case of default. The extra collateral helps to cover the loan in case the loan value exceeds the value of collateral.
+      key: C
+    - content: None of these
+      key: D
+
+- uuid: 9b09ff86-e6d3-4861-987d-aa819fe5622a
+  type: MultipleChoice
+  content: What are the use cases of lending and borrowing?
+  hint: NoHint
+  explanation: Lending increases the value of assets by gaining interest on the lent assets and borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient. Borrowing increases an investment's potential return through leveraging.
+  answerKeys:
+    - A
+    - C
+    - D
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Lending increases the value of assets by gaining interest.
+      key: A
+    - content: Lending improves decentralization and security.
+      key: B
+    - content: Borrowing helps you to get money in urgent situations or for improvement purposes when capital is insufficient.
+      key: C
+    - content: Borrowing increases an investment's potential return through leveraging.
+      key: D
+- uuid: a4ea1fb0-f2dd-4fba-b3b3-75e57164010c
+  type: SingleChoice
+  content: The act of allowing someone else to use ones assets is called?
+  hint: Lending
+  explanation: Lending is an action of allowing a person or an organization to use the sum of money under an agreement to pay it back later with remuneration, usually in the form of interest. 
+  answerKeys:
+    - B
+  subTopics:
+    - why-do-we-lending-borrowing
+  difficultyLevel: Low
+  choices:
+    - content: Leveraging
+      key: A
+    - content: Lending
+      key: B
+    - content: Borrowing
+      key: C
+    - content: Banking
+      key: D
+
+- uuid: dcfa8605-e19c-41e3-908f-8ab9087d4182
+  type: SingleChoice
+  content: How DeFi's are obtaining realtime data like asset prices, since blockchains are isolated and cannot talk to the web?
+  hint: Oracle
+  explanation: Oracles are nodes which provides realtime data like asset prices to the blockchains.
   answerKeys:
     - B
   subTopics:
     - why-do-we-lending-borrowing
-  difficultyLevel: "Low"
+  difficultyLevel: Medium
+  choices:
+    - content: Blockchains get real world data from internet
+      key: A
+    - content: Through oracles
+      key: B
+    - content: They can't get real world data
+      key: C
+    - content: None of the above
```

Comment:
replace `None of the above` with some other option as we already have `They can't get real world data`